src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs.cljs

```diff
@@ -99,9 +101,12 @@
                                             (react/call :load-method-configs this)
                                             (swap! state update-in [:method-confs]
                                               assoc (:selected-index @state) new-conf)))}]
-         (:method-confs-loaded? @state) [WorkspaceMethodsConfigurationsList {:workspace (:workspace props)
-                                                                             :method-confs (:method-confs @state)
-                                                                             :parent-state state}]
+         (:method-confs-loaded? @state) [WorkspaceMethodsConfigurationsList
+                                          {:on-import
+                                             (fn [] (react/call :load-method-configs this))
```

Comment:
I think our codebase should be mostly unaware that mock data exists, so if the only reason to include a parameter is to make it look better when using mock data, my preference would be to not include it. Otherwise, I might look at this code later and say, "Why does the config get passed in? That seems less safe than refreshing the list to make sure the import actually went through."

If you decide it is worth doing, then I would at least ask for a comment calling out that the parameter is only used for mock data/demo purposes. My hope, though, is that we will never have to use mock data for demos again. Maybe that's naively optimistic :)